doc/user/content/release-notes.md

```diff
@@ -113,6 +113,12 @@ changes that have not yet been documented.
   Previous versions of Materialize would assume the head time unit
   to always be hours. The new behavior matches PostgreSQL.
 
+- **Breaking change.** In Kafka sources, SSL configurations for the Confluen
+  Schema Registry no longer default to the Kafka options. To experience the old behavior, use
+  `ssl_follow_kafka = true`. See
+  [Confluent Schema Registry options](/sql/create-source/avro-kafka#confluent-schema-registry-options)
+  for details.
```

Comment:
This needs to go above the line that says "put new release notes above this line."

Reply:
I mess this up EVERY time